requirements.txt

```diff
@@ -4,4 +4,6 @@ pytest
 pycryptodome
 pytest-asyncio
 click
-base58
\ No newline at end of file
+base58
+pymultihash
+multiaddr
```

Comment:
Please add multihash as a dependency.

Reply:
I already added the multihash lib I used, it is pymultihash, line 8

Reply:
My mistake, thank you!